runtime/udp.cpp

```diff
@@ -97,12 +105,14 @@ static Stream udp_stream_socket_client(const string &url, int64_t &error_number,
   sock_fd = socket(AF_INET, SOCK_DGRAM, 0);
   if (sock_fd == -1) {
     dl::leave_critical_section();
-    RETURN_ERROR(-4, "Can't create udp socket");
+    set_error(-4, "Can't create udp socket");
+    return quit(sock_fd, Optional<string>(string("System call \'socket(...)\' got error \'").append(strerror(errno)).append("\'")));
```

Comment:
`Optional<T>` is implicitly convertible of `T`
```suggestion
    return quit(sock_fd, string("System call \'socket(...)\' got error \'").append(strerror(errno)).append("\'"));
```

Also can we write single quotes without slash?

Reply:
fix